File: program/program/convert-days-to-hours/ConvertDaysToHours.js
function convertDayToHours(days) {
    // 1 days = 24hours
    // 5 days = ? hours
    // 5 days = 5 * 24 = 120 hours

    return days * 24;
}

console.log(`${convertDayToHours(5)} Hours`)

File: program/program/convert-days-to-hours/ConvertDaysToHours.cs
using System;

class MainClass {
    public static void Main (string[] args) {
    Console.WriteLine ("Please enter the days :");
    double days = Convert.ToDouble(Console.ReadLine());
    double hours = days * 24;
    Console.WriteLine(days+ " days are "+ hours + " hours");
  }
}
